### DISCUSSION TITLE
Update to 2.4.2

### DISCUSSION THREAD

**Discussion by vadimkhm** [devlooped/xunit.assemblyfixture#18] [General]
Hi @kzu, it would be great to have to update all dependencies to version 2.4.2, is it possible to have it soon?

Thanks,
Vadim

**Comment by kzu** [comment]
I don't think this package actually would break if you use 2.4.2, does it?

**Reply by vadimkhm** [reply]
Actually, I received the warning about different versions, but I solved the problem by forking the repo, of course, it will be better to use the library directly, but if this is a problem to update, no worry I will use the forked version.
Thanks

**Reply by kzu** [reply]
Shipped as 2.2.0

**Reply by vadimkhm** [reply]
Thank you very much!
